app/models/c2_cancel_attributes.rb

```diff
@@ -0,0 +1,32 @@
+class C2CancelAttributes
+  def initialize(auction)
+    @auction = auction
+  end
+
+  def to_h
+    {
+      status: 'cancelled',
```

Comment:
Ha, the source seems to be `canceled` so you are right. Filed a PR to fix the docs